src/services/product.js

```diff
@@ -0,0 +1,36 @@
+const { Product } = require('../config');
+
+class ProductService {
+  async create(productData) {
+    const product = Product.build(productData);
+    await product.save();
+    return product;
+  }
+
+  find() {
+    return Product.findAll();
+  }
```

Comment:
this function needs the 'async' next to the funct. name `find()` and the 'await' on the `return Product.findAll()` or those are unnecessary for this one?

Reply:
Thank for your comment @ACarrascoH !
This returns a promise, so we can await the 'first one', like
`await productService.find() `=> this will call `Product.findAll()` => so when we await, we are awaiting 'all' 
<img width="538" alt="image" src="https://github.com/user-attachments/assets/7a8f9a0d-e596-4de8-ad61-27ce7feee1cf">

Reply:
Ooooh, ok i understand